crates/sncast/src/starknet_commands/account/import.rs

```diff
@@ -196,6 +204,15 @@ fn get_private_key_from_input() -> Result<Felt> {
     parse_input_to_felt(&input)
 }
 
+fn generate_default_account_name() -> String {
+    let start = SystemTime::now();
+    let since_epoch = start
+        .duration_since(UNIX_EPOCH)
+        .expect("Time went backwards");
+    let timestamp = since_epoch.as_millis();
+    format!("account-{}", timestamp)
+}
+
```

Comment:
Let's refactor this function to create account with sequential names (`account-1`, `account-2`, ...). Name with timestamp (e.g. `account-1729865041172`) is too complicated.
Our logic should find next available numer (and handle gaps, e.g. having `account-1` and `account-3` in file, imported account should receive name `account-2`).

You can see how accounts are read from the file [here](https://github.com/foundry-rs/starknet-foundry/blob/e5918db054d7264fd08e79dee1c33d6391009eda/crates/sncast/src/starknet_commands/account/mod.rs#L102).

Reply:
Alright. This was my initial plan. I'll fall back to it.